src/app.js

```diff
@@ -1 +1,33 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const copyFile = () => {
+  const args = process.argv.slice(2);
+
+  if (args.length !== 2) {
+    console.log('No files for copy');
+
+    return;
+  }
+
+  const filePath = args[0];
+  const copiedFilePath = args[1];
+
+  if (filePath === copiedFilePath) {
+    console.log('Paths of files are the same');
+
+    return;
+  }
+
+  try {
+    const data = fs.readFileSync(filePath);
```

Comment:
This way you are loading the **entire** file into memory (data constant). Program will eat a lot of memory and take a long time.

You can test this copying files that are several GBs in size.

Instead, there is a `fs.copyFileSync` in node that lets you avoid this ([check the docs](https://nodejs.org/api/fs.html)).